eng/common/scripts/common.ps1

```diff
@@ -45,3 +45,4 @@ $GetDocsMsDevLanguageSpecificPackageInfoFn = "Get-${Language}-DocsMsDevLanguageS
 $GetGithubIoDocIndexFn = "Get-${Language}-GithubIoDocIndex"
 $FindArtifactForApiReviewFn = "Find-${Language}-Artifacts-For-Apireview"
 $TestProxyTrustCertFn = "Import-Dev-Cert-${Language}"
+$ValidateDocsMsPackagesFn = "Validate-${Language}-DocMsPackages"
```

Comment:
Is there any further common logic we should pull into the eng/common scripts here, like the calling of the docker image assuming the parameters are the same.

Reply:
Call docker image is only one liner code. besides we use different parameter for different language.
Like Java use group id, Python use extra-index-url etc

Reply:
Ok The call is one line of code but I know you are also handling various exit codes in multiple places. However if they all have different parameters they pass to the docker image then I agree there isn't much we can share.